### PR TITLE
allow to set security token by env variable

### PR DESCRIPTION
**Pull request by cgroschupp**

### CODE REVIEW

File: opentelekomcloud/provider.go [hunk 1]
```diff
@@ -94,6 +94,7 @@ func Provider() terraform.ResourceProvider {
 			"security_token": {
 				Type:        schema.TypeString,
 				Optional:    true,
+				DefaultFunc: schema.EnvDefaultFunc("OS_SECURITY_TOKEN", ""),
```
Comment (lego963) [inline opentelekomcloud/provider.go:97]: mb let's rename to `OS_TOKEN`?
https://wiki.openstack.org/wiki/CLIAuth

**Comment by lego963** [comment]
@cgroschupp, I was wrong, security token is something another
https://github.com/terraform-provider-openstack/terraform-provider-openstack/blob/master/openstack/provider.go#L102
OS_TOKEN == OS_AUTH_TOKEN
Revert last change :)

**Review by lego963 — APPROVED** [review]

**Review by outcatcher — APPROVED** [review]

**Comment by outcatcher** [comment]
@cgroschupp Your branch is out-of-date with the base. Can you please update your branch?

**Comment by cgroschupp** [comment]
@outcatcher done

**Pull request merged**